ddtrace/_monkey.py

```diff
@@ -212,98 +219,35 @@ def patch(raise_errors=True, patch_modules_prefix=DEFAULT_MODULES_PREFIX, **patc
 
         >>> patch(psycopg=True, elasticsearch=True)
     """
-    modules = [m for (m, should_patch) in patch_modules.items() if should_patch]
-    for module in modules:
-        if module in _PATCH_ON_IMPORT:
-            modules_to_poi = _PATCH_ON_IMPORT[module]
-            for m in modules_to_poi:
-                # If the module has already been imported then patch immediately
-                if m in sys.modules:
```

Comment:
Checking my understanding: this condition is removed because your removal of cross-contrib imports means that it will never be True. Is that right?

Reply:
Correct! If the module is already imported `when_imported` will trigger the hook immediately anyway.